src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerRegenerateCourtOrders.java

```diff
@@ -136,4 +131,44 @@ public SubmittedCallbackResponse submitted(CaseDetails<CaseData, State> details,
         notificationDispatcher.send(regenerateCourtOrdersNotification, caseData, details.getId());
         return SubmittedCallbackResponse.builder().build();
     }
+
+    private void removeExistingAndGenerateNewFinalOrderGrantedCoverLetters(CaseData caseData, long caseId) {
+
+        final List<DocumentType> documentTypesToRemove =
+            List.of(FINAL_ORDER_GRANTED_COVER_LETTER_APP_1, FINAL_ORDER_GRANTED_COVER_LETTER_APP_2);
```

Comment:
now that we only pre-generate the final order do we actually need to regenerate the coverletter - wouldn't that happen automatically now?